src/main/java/graphql/language/StringValue.java

```diff
@@ -110,8 +113,8 @@ public StringValue transform(Consumer<Builder> builderConsumer) {
     }
 
     public static final class Builder implements NodeBuilder {
-        private SourceLocation sourceLocation;
-        private String value;
+        private @Nullable SourceLocation sourceLocation;
+        private @Nullable String value;
```

Comment:
Similar to above Value classes, I will double check if value can be nullable or if in practice it is represented by a `NullValue`